doc/source/release-notes/Percona-Server-8.0.19-10.rst

```diff
@@ -0,0 +1,63 @@
+.. _PS-8.0.19-10:
+
+================================================================================
+*Percona Server for MySQL* 8.0.19-10
+================================================================================
+
+:Date: March 23, 2020
+
+:Installation: `Installing Percona Server for MySQL <https://www.percona.com/doc/percona-server/8.0/installation.html>`_
+
+`Percona Server for MySQL <https://www.percona.com/software/mysql-database/percona-server>`_ 8.0.19-10
+includes all the features and bug fixes available in
+`MySQL 8.0.19 Community Edition <https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-19.html>`_
+in addition to enterprise-grade features developed by Percona.
+
+New Features
+================================================================================
+
+* :jirabug:`PS-4464`: Exposed the last global transaction identifier (GTID) executed for a CONSISTENT SNAPSHOT.
+
+
+
+Improvements
+================================================================================
+
+* :jirabug:`PS-6775`: Removed the KEYRING_ON option from the default_table_encryption.
+* :jirabug:`PS-6733`: Added binary search to the Data masking plugin.
+
```

Comment:
Please add here PS-5729 here, I have forgotten to set status to fixed ...

Reply:
Verified status is fixed. Thanks, Robert.